src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -176,13 +176,12 @@ public void TestStartWithBadWorkingDirectory()
         [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.HasWindowsShell))]
```

Comment:
Do you not want this to run on Unix anymore? Because by changing HasWindowsShell to only be true on Windows, it will stop running on Unix.

Reply:
Oh I see, you broke out a new Unix one.